docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -121,77 +120,77 @@ The NEB part of CASTEP should pick up on the imposed ionic constraint(s) and you
 
 ### Restarting / Continuing 
 
 Sometimes the calculation will run out of time/number of iterations before the NEB calculation is finished. 
 You will probably see a "Failed to converge" message in the .castep output in that case; always read through your output files!
+*[Not obvious in .ts file? Shouldn't need to positively check for unconverged results.]*
 
-If you simply re-run the calculation without changing anything, it will simply start from the beginning again -- probably not what you want!
+If you re-run the calculation without changing anything, it will simply start from the beginning again -- probably not what you want!
 
-In order restart the NEB from the last checkpointed state, you need to **explicitly** set the name of the .check file you want to continue from. In our example that would be:
+In order restart the NEB from the last checkpointed state, you need to **explicitly** set the name of the .check file you want to continue from. *[In .param?]* In our example that would be:
 
 ```
 continuation: nh3.check
 ```
 
-In the .castep output you would then see this message: `Coordinates of NEB images loaded from checkpoint file`. 
+In the .castep output you would then see: `Coordinates of NEB images loaded from checkpoint file`. 
 
 !!! warning
-      Unlike other parts of CASTEP, setting `continuation: default` does not seem to work for continuing/restarting NEB calculations! You must explicitly set the name of checkpoint file.
+      Unlike other CASTEP tasks, setting `continuation: default` does not seem to work for continuing/restarting NEB calculations! You must explicitly set the name of checkpoint file.
 
 
 
 
 ## Analysis
 
-Once you have a converged NEB, you can use the provided Python utility: readTS to parse and analyse the NEB. 
-You first need to make sure the readTS module is on your PYTHONPATH. You can do this by going to your CASTEP source directory, and in `castep/Utilities/readts` you will find a setup.py script. Running `python setup.py` should install the module to your path. You can also manually add it by: 
+Once you have a converged NEB, the provided Python utility `readTS` can be used to parse and analyse the NEB. 
+To add the readTS module to your PYTHONPATH, you can find a setup.py script in the `castep/Utilities/readts` directory of your CASTEP source directory. Running `python setup.py` should install the module to your path. You can also manually add it by: *[Add to what? .bashrc or current (bash) console?]* 
 
 ``` bash
 export PYTHONPATH="/path/to/castep/Utilities/readts:$PYTHONPATH"
 ```
 
-(changing the `/path/to/castep/` bit to wherever CASTEP is on your machine!).
+(changing the `/path/to/castep/` bit to wherever CASTEP is on your machine).
 
-You can then extract the NEB path using something like this python code:
+You can then extract the NEB path using something like this python code:  *[This looks sufficiently general that you could supply the analysis.py script?]*
```

Comment:
It's provided in the tar file with the tutorial files and the full script is shown here -- do we also need a separate link to download .py file?

Reply:
OK. But then there's not much point in giving the source of analysis.py in the tutorial. I would rename analysis to analysisTS or something more meaningful, and refer to reader to the script in the tutorial bundle. it would be more useful in the tutorial to highlight anything that the user might want to tweak / customise.